### PR TITLE
 Add /package/describe to the example

### PR DESCRIPTION
**Pull request by bamarni**

### CODE REVIEW

File: openapi/dcos.yaml [hunk 1]
```diff
@@ -2163,9 +2163,10 @@ components:
       properties:
         command:
           $ref: '#/components/schemas/command'
-        config:
-          additionalProperties: true
-          type: object
+        # Causes a JSON unmarshal error at the moment, config should be defined as a JSON schema instead.
+        # config:
```
Comment (jimmidyson) [inline openapi/dcos.yaml]: In all our uses so far, this is a critical property of the response: this is the only place it is available, not via package search response, etc. I wouldn't want to see this merged without this being available via this call.

Comment (jimmidyson) [reply openapi/dcos.yaml]: This works if you remove the `additionalProperties: true` from `config` declaration (generated code is `map[string]interface{}` which works great for now.

File: openapi/dcos.yaml [hunk 2]
```diff
@@ -2163,10 +2163,9 @@ components:
       properties:
         command:
           $ref: '#/components/schemas/command'
-        # Causes a JSON unmarshal error at the moment, config should be defined as a JSON schema instead.
-        # config:
-        #   additionalProperties: true
-        #   type: object
+        config:
+          additionalProperties: false
```
Comment (jimmidyson) [inline openapi/dcos.yaml]: You can actually remove it completely as it is with other properties.

Comment (bamarni) [reply openapi/dcos.yaml]: I've added a comment as it should be defined as a JSON schema eventually.

File: openapi/dcos.yaml (hunk 1 above)
Comment (bamarni) [reply openapi/dcos.yaml]: It worked out indeed :+1:

**Pull request merged**